core/invehicle-digital-twin/src/main.rs

```diff
@@ -51,9 +52,14 @@ async fn register_invehicle_digital_twin_service_with_chariott(
     chariott_uri: &str,
     invehicle_digital_twin_uri: &str,
 ) -> Result<(), Status> {
-    let mut client = ServiceRegistryClient::connect(chariott_uri.to_string())
-        .await
-        .map_err(|e| Status::internal(e.to_string()))?;
+    let uri = utils::get_uri(chariott_uri).map_err(|err| {
+        Status::failed_precondition(format!(
+            "Unable to get container environment variable with error: {err}"
```

Comment:
Perhaps include the environment variable's name?

Reply:
Updated to add the env variable name in get_uri.